tests/phpunit/Event/Subscriber/MutationTestingConsoleLoggerSubscriberTest.php

```diff
@@ -226,20 +227,20 @@ public function test_it_outputs_escaped_mutants_when_mutation_testing_is_finishe
 
     public function test_it_does_not_output_escaped_mutants_when_mutation_testing_is_finished_with_no_escaped_mutants(): void
     {
+        $matcher = $this->atLeastOnce();
         $this->output
-            ->expects($this->atLeastOnce())
+            ->expects($matcher)
             ->method('writeln')
-            ->withConsecutive(
-                [
-                    [
+            ->willReturnCallback(function (mixed $value) use ($matcher): void {
+                match ($matcher->numberOfInvocations()) {
+                    1 => $this->assertSame([
                         '',
                         '',
-                    ],
-                ],
-                [
-                    '<options=bold>0</options=bold> mutations were generated:',
-                ]
-            );
+                    ], $value),
+                    2 => $this->assertSame('<options=bold>0</options=bold> mutations were generated:', $value),
+                    default => 'noop'
+                };
```

Comment:
It is kinda sad we have to do this on PHPUnit 10.

Reply:
integrated community replacement that helps to avoid it